### PR TITLE
Fix inappropriate usages of xml tags in Az.Accounts

### PR DESCRIPTION
**Pull request by BethanyZhou**
## Description
To resolve https://github.com/Azure/autorest.powershell/issues/843

## Checklist

- [ ] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [ ] The title of the PR is clear and informative
- [ ] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [ ] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - {Please put the link here}
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

File: src/Accounts/Accounts/Common/AzureContextModificationCmdlet.cs [hunk 1]
```diff
@@ -188,6 +188,7 @@ protected  bool TryGetExistingContextNameParameter(string name, out RuntimeDefin
         /// Generate a runtime parameter with ValidateSet matching the current context
         /// </summary>
         /// <param name="name">The name of the parameter</param>
+        /// <param name="parameterSetName"></param>
```
Comment (BethanyZhou) [inline src/Accounts/Accounts/Common/AzureContextModificationCmdlet.cs:191]: Add param tag for all parameters in functions.

File: src/Accounts/Accounts/CommonModule/AzModule.cs [hunk 2]
```diff
@@ -117,7 +117,7 @@ public void OnModuleLoad(string resourceId, string moduleName, PipelineChangeDel
         /// <param name="exception">The <see cref="System.Exception" /> that is being thrown (if available)</param>
         public async Task EventListener(string id, CancellationToken cancellationToken, GetEventData getEventData, SignalDelegate signal, InvocationInfo invocationInfo, string parameterSetName, string correlationId, string processRecordId, System.Exception exception)
         {
-            /// Drain the queue of ADAL events whenever an event is fired
+            // Drain the queue of ADAL events whenever an event is fired
```
Comment (BethanyZhou) [inline src/Accounts/Accounts/CommonModule/AzModule.cs:120]: /// is a xml tag, here should be a comment

File: src/Accounts/Accounts/CommonModule/ContextAdapter.cs [hunk 3]
```diff
@@ -127,9 +127,9 @@ public void OnNewRequest(InvocationInfo invocationInfo, string correlationId, st
...
         /// <param name="invocationInfo">The <see cref="System.Management.Automation.InvocationInfo" /> from the cmdlet</param>
         /// <param name="correlationId">The <see cref="string" /> containing the correlation id for the cmdlet (if available)</param>
-        /// <param name="resourceTypes">An <see cref="System.String[]"/> containing resource (or resource types) being completed  </param >
-        /// <param name="parentResourceParameterNames"> An <see cref="System.String[]"/> containing list of parent resource parameter names (if applicable)</param >
-        /// <returns>A <see cref="System.String[]"/> containing the valid options for the completer.</returns>
+        /// <param name="resourceTypes">An <see cref="System.String"/>[] containing resource (or resource types) being completed  </param >
+        /// <param name="parentResourceParameterNames"> An <see cref="System.String"/>[] containing list of parent resource parameter names (if applicable)</param >
+        /// <returns>A <see cref="System.String"/>[] containing the valid options for the completer.</returns>
```
Comment (BethanyZhou) [inline src/Accounts/Accounts/CommonModule/ContextAdapter.cs:132]: can't find type System.String[]

File: src/Accounts/Accounts/CommonModule/EventData.cs [hunk 4]
```diff
@@ -25,13 +25,13 @@ namespace Microsoft.Azure.Commands.Common
...
     using PipelineChangeDelegate = Action<EventData>;
-
-    [TypeConverter(typeof(EventDataConverter))]
+    
     ///	<remarks>
     /// In PowerShell, we add on the EventDataConverter to support sending events between modules.
     /// Obviously, this code would need to be duplcated on both modules.
     /// This is preferable to sharing a common library, as versioning makes that problematic.
     /// </remarks>
+    [TypeConverter(typeof(EventDataConverter))]
```
Comment (BethanyZhou) [inline src/Accounts/Accounts/CommonModule/EventData.cs:34]: Tag should above attribute

File: src/Accounts/Accounts/CommonModule/VTable.cs [hunk 5]
```diff
@@ -42,12 +42,16 @@ public class VTable
...
+        /// <code>
+        /// public object GetParameterValue(string resourceId, string moduleName, System.Management.Automation.InvocationInfo invocationInfo, string name)
+        /// </code>
+        /// <para>resourceId: <c>string</c>containing the expected resource id (ie, ARM).</para>
+        /// <para>moduleName: <c>string</c>containing the name of the module being loaded.</para>
+        /// <para>invocationInfo: The <see cref="System.Management.Automation.InvocationInfo" /> from the cmdlet</para>
+        /// <para>correlationId: The <see cref="string" /> containing the correlation id for the cmdlet</para>
+        /// <para>name: The <see cref="string" /> parameter name being asked for</para>
+        /// </example>
         public GetParameterDelegate GetParameterValue;
```
Comment (BethanyZhou) [inline src/Accounts/Accounts/CommonModule/VTable.cs:56]: There are no parameters called resourceId,moduleName, etc. Move them to example block

**Review by BethanyZhou — COMMENTED** [review]
No change log is expected as there are only XML tag changes

**Review by wyunchi-ms — APPROVED** [review]

**Pull request merged**